aiosmtpd/smtp.py

```diff
@@ -78,13 +97,38 @@ def max_command_size_limit(self):
             return self.command_size_limit
 
     def connection_made(self, transport):
-        super().connection_made(transport)
-        self.peer = transport.get_extra_info('peername')
-        self.transport = transport
-        log.info('Peer: %s', repr(self.peer))
-        # Process the client's requests.
-        self.connection_closed = False
-        self._handler_coroutine = self.loop.create_task(self._handle_client())
+        if(self.transport is not None
+                and isinstance(transport, sslproto._SSLProtocolTransport)):
+            # It is STARTTLS connection over normal connection
+            self._reader._transport = transport
+            self._writer._transport = transport
+            self.transport = transport
+            # Reset state due to rfc3207 part 4.2
+            self._set_rset_state()
+            self.seen_greeting = False
```

Comment:
Why set `seen_greeting` here to False when the constructor already sets it to the empty string?